packages/core/lib/src/widgets/tweet/retweets.dart

```diff
@@ -1,4 +1,3 @@
-import 'package:core/core.dart';
 import 'package:core/src/styles/app_colors.dart';
 import 'package:core/src/styles/twitter_icons.dart';
 import 'package:core/src/widgets/tweet/tweet_metric.dart';
```

Comment:
Interesting, that you removed this line. I prefer general imports over detailed imports, especially if it comes to importing from (kind of) external packages.

Reply:
Agreed. Optimized imports to use `package:core/core.dart` 👍